ui/src/app/applications/components/application-urls.tsx

```diff
@@ -61,7 +61,9 @@ export const ApplicationURLs = ({urls}: {urls: string[]}) => {
                         e.stopPropagation();
                         window.open(externalLinks[0].ref);
                     }}>
-                    <i className='fa fa-external-link-alt' />{' '}
+                    <a href='#'>
```

Comment:
per line 58 above, this is already inside an anchor element. nesting anchors inside anchors is not semantically correct (nor is `href='#'`)